server/db/admin.js

```diff
@@ -0,0 +1,29 @@
+import bcrypt from "bcryptjs";
+import db from "./index";
+
+const fullname = 'Admin Fastfood';
```

Comment:
Please move all admin details to your environment

Reply:
Okay... I thought as much too
I'd do that now